cmd/pebble/main.go

```diff
@@ -340,7 +340,10 @@ func run() error {
 	if pebbleDir == "" || !osutil.IsDir(pebbleDir) {
 		return fmt.Errorf("$PEBBLE must point to a pebble directory")
```

Comment:
This here needs to allow pebbleDir to be unspecified (otherwise a default is pointless).

Reply:
Good point, thanks.